test/parallel/test-tick-processor.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+var fs = require('fs');
+var assert = require('assert');
+var path = require('path');
+var cp = require('child_process');
+var common = require('../common');
+
+common.refreshTmpDir();
+
+process.chdir(common.tmpDir);
+cp.execFileSync(process.execPath, ['-prof', '-pe',
+    'function foo(n) {' +
+    'require(\'vm\').runInDebugContext(\'Debug\');' +
+    'return n < 2 ? n : setImmediate(function() { foo(n-1) + foo(n-2);}); };' +
+    'setTimeout(function() { process.exit(0); }, 2000);' +
+    'foo(40);']);
+var matches = fs.readdirSync(common.tmpDir).filter(function(file) {
+  return /^isolate-/.test(file);
+});
+if (matches.length != 1) {
+  assert.fail('There should be a single log file.');
+}
+var log = matches[0];
+var processor =
+    path.join(common.testDir, '..', 'tools', 'v8-prof', getScriptName());
+var out = cp.execSync(processor + ' ' + log, {encoding: 'utf8'});
+assert(out.match(/LazyCompile.*foo/));
+if (process.platform !== 'win32' &&
+    process.platform !== 'sunos' &&
+    process.platform !== 'freebsd') {
+  assert(out.match(/RunInDebugContext/));
+}
```

Comment:
I'm not sure what skip output is. The *foo assert checks for JS symbols and the RunInDebugContext checks for C++ symbols. For windows, smartos, and freebsd, the C++ symbols are not being mapped so the test fails for now. Is there a better way to specify a test that should only pass on some OSes?